torch/_inductor/config.py

```diff
@@ -854,6 +854,14 @@ class cpp:
         == "1"
     )
 
+    # Use ffp-contract=fast when compiling
+    enable_floating_point_contract_fast_flag = (
```

Comment:
Makes sense in general, but can we combine with the existing `enable_floating_point_contract_flag`? Adding another one can be confusing.